src/public/modules/forms/services/spcp-session.client.factory.js

```diff
@@ -40,13 +40,6 @@ function SpcpSession($interval, $q, Toastr, $window, $cookies) {
       session.userName = undefined
     },
     logout: function (authType) {
-      $cookies.remove(
-        // TODO (#2329): To remove after old cookies have expired
-        session.cookieNames[authType],
```

Comment:
Should we also remove the `session.cookieNames` declaration too since that is now unused? (line 21-25)

Reply:
thanks! done